news.py

```diff
@@ -36,9 +43,23 @@ def search_articles():
 
     return render_template("results.html", url_title_list=url_title_list, query=query)
 
+@app.route("/login", methods=['POST'])
+def login():
+    username = request.form['username']
+    password = request.form['password']
+    print(username)
```

Comment:
do we want to have print statements in the application? maybe we should use logging instead